scripts/new_major_version.go

```diff
@@ -0,0 +1,103 @@
+// go run scripts/new_major_version.go -ver v5 -name tls goes part of the way to udpate all Go source references to a
```

Comment:
Calling this out. Doesn't really belong here, but I was trying to see if it can automate part of the edits for a major version release via the "gofmt" route. This is the "regex based" alternative to https://github.com/pulumi/ci-mgmt/pull/319